packages/terra-functional-testing/src/commands/utils/github.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * This module encapsulates the full-power octokit library inside a simple interface.
+ * https://github.com/octokit/octokit.js
+ *
+ * It supports both github.com and enterprise github hosts.
+ * Requires your github host to have API v3 and https.
+ */
+const { Octokit } = require('octokit');
+
+/**
+ * The info needed to look up a PR.
+ * @typedef PrLookupInfo
+ * @type {object}
+ * @property {string} githubHostname
+ * @property {string} repoOwner
+ * @property {string} repoName
+ * @property {string} prNumber
+ */
+
+/**  Info about a PR, fetched from a github instance. */
+class PrInfo {
```

Comment:
What is the reason for making this a class? Is it because we plan to add functions to it in the future?

Reply:
No. I made it a class merely so that it could serve as a datatype that consumers can easily know the structure of.

Reply:
Oh wait: yes kind of :).  I'm going to re-work this class as an adapter - it will take in the raw return JSON from the octokit API but expose only the info we know we need. Like a simplification filter. This will keep the rest of our code from having to know the github ReST API JSON. I'll need to add one for @BenBoersma's code as well.

Reply:
Ok. At first I was thinking that `lookupPR` could just return an object with these attributes instead of creating a class for it.